toolchain/check/import_ref.cpp

```diff
@@ -413,6 +413,29 @@ class ImportRefResolver {
     return context_.inst_blocks().AddCanonical(contents);
   }
 
+  auto GetLocalGenericInstanceArgs(SemIR::GenericInstanceId instance_id)
+      -> llvm::SmallVector<SemIR::InstId> {
+    if (!instance_id.is_valid()) {
+      return {};
```

Comment:
Minor, but given the vector return, do you want a single return variable for rvo?

Reply:
It's unnecessary in this case -- both return statements construct the `SmallVector` in place, so we get guaranteed RVO here and don't need to rely on NRVO.